tests/unit/h/conftest.py

```diff
@@ -240,6 +241,12 @@ def with_auth_client(auth_client, pyramid_config):
     )
 
 
+@pytest.fixture
+def frozen_time():
+    with freeze_time("2012-01-14 03:21:34") as frozen_time_factory:
```

Comment:
Does the particular time matter here for any reason?

Reply:
No this is just an arbitrary time